client/src/luigi-client.js

```diff
@@ -443,6 +448,52 @@ const LuigiClient = {
           promises.confirmationModal.rejectFn = reject;
         });
         return promises.confirmationModal.promise;
+      },
+
+      /**
+       * Shows an alert.
+       * @param {Object} settings the settings for the alert
+       * @param {string} settings.text the content of the alert. To add a link to the content, you have to setup the link in `links` object. The key(s) in the `links` object must be used in the text to reference the links, wrapped in curly braces with no spaces. If no text is specified, the alert will not be displayed 
+       * @param {('info'|'success'|'warning'|'error')} settings.type sets the type of the alert
+       * @param {Object} settings.links provides links data
+       * @param {Object} settings.links.LINK_KEY object containing the data for a concrete link.  
+       * Read above on how to use the key to for the link to be properly rendered in the alert message.
```

Comment:
```suggestion
       * To properly render the link in the alert message refer to the description of the **settings.links.LINK.KEY** parameter.
```

Reply:
Almost, it is `settings.text` instead of `settings.links.LINK.KEY`. I will commit it separately.